CHANGELOG.md

```diff
@@ -6,7 +6,12 @@ The format is based on [Keep a
 Changelog](http://keepachangelog.com/en/1.0.0/) and this project
 adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## [Unreleased] - xxxx-xx-xx
+## [v1.1.0] - 2018-11-07
+### Added
+- New endpoints [#88](https://github.com/sharetribe/flex-sdk-js/pull/88):
+  * 'own_listings/create_draft'
+  * 'own_listings/publish_draft'
+  * 'own_listings/discard_draft'
```

Comment:
Users of the SDK most likely are using the instance methods, maybe listings those would be a bit nicer? For example:

```js
  * `sdk.ownListings.createDraft(/* ... */)`
  * `sdk.ownListings.publishDraft(/* ... */)`
  * `sdk.ownListings.discardDraft(/* ... */)`
```

You could also link to the corresponding API docs to get more info of the params etc.